src/services/RedocNormalizedOptions.ts

```diff
@@ -189,6 +206,7 @@ export class RedocNormalizedOptions {
   disableSearch: boolean;
   onlyRequiredInSamples: boolean;
   showExtensions: boolean | string[];
+  sideNavStyle: string;
```

Comment:
But would be awesome to use string enums here: https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

```suggestion
  sideNavStyle: 'summary-only' | 'path-only'
```

Reply:
@RomanHotsiy 
Actually, I didn't find string enums in the project. Could you suggest a good location to store enum definition?

Reply:
Or, just your suggestion is enough?

Reply:
Hmm, keep it as my suggestion for now. We'll add enums later.

Reply:
@RomanHotsiy It looks ugly, so I applied string enums. Please take another look. Thanks!